lib/evmone/instructions.hpp

```diff
@@ -1088,6 +1088,25 @@ inline code_iterator retf(StackTop /*stack*/, ExecutionState& state, code_iterat
     return p;
 }
 
+inline code_iterator jumpf(StackTop stack, ExecutionState& state, code_iterator pos) noexcept
+{
+    const auto index = read_uint16_be(&pos[1]);
+    const auto& header = state.analysis.baseline->eof_header;
+    const auto stack_size = &stack.top() - state.stack_space.bottom();
+
+    const auto callee_required_stack_size =
+        header.types[index].max_stack_height - header.types[index].inputs;
+    if (stack_size + callee_required_stack_size > StackSpace::limit)
+    {
+        state.status = EVMC_STACK_OVERFLOW;
+        return nullptr;
+    }
+
+    const auto offset = header.code_offsets[index] - header.code_offsets[0];
+    auto code = state.analysis.baseline->executable_code;
```

Comment:
Not obvious to me why this isn't `const`.

Reply:
Looks like it can be, fixed.